packages/core/src/demo/utils/stackblitz/react.ts

```diff
@@ -66,7 +65,10 @@ class App extends React.Component {
     ></ ${demo.chartType.vanilla}>
   )
 }
-ReactDOM.render(<App />, document.getElementById('root'))
+const root = ReactDOM.createRoot(document.getElementById('root'))
+root.render(
+  <App />
+)
```

Comment:
This makes the StackBlitz React examples use React 18 (createRoot) behavior.

Reply:
yes I was going to do this!